src/commands/fun/poll.ts

```diff
@@ -1,7 +1,24 @@
-import {MessageEmbed, Message, User} from "discord.js";
+import {MessageEmbed, Message, User, MessageActionRow, MessageButton} from "discord.js";
 import {NamedCommand, RestCommand, poll, CHANNEL_TYPE, SendFunction, Command} from "onion-lasers";
+import {SlashCommandBuilder} from "@discordjs/builders";
+import {CommandInteraction} from "discord.js";
 import {pluralise} from "../../lib";
 
+export const header = new SlashCommandBuilder()
+    .setDescription("Create a poll.")
+    .addStringOption((option) => option.setName("question").setDescription("Question for the poll").setRequired(true))
+    .addIntegerOption((option) =>
+        option.setName("duration").setDescription("Duration of the poll in seconds.").setRequired(false)
```

Comment:
This description is wrong, as the bot actually takes the duration in *milliseconds.*
```js
const duration = options.getInteger("duration", false);
execSlashPoll(interaction, question, duration || 60000);
```
Please do a manual conversation of user input (e.g. `20` seconds) to milliseconds.

Reply:
Probably should put `parseDuration` elsewhere and use it for consistency.
https://github.com/keanuplayz/TravBot-v3/blob/e7217cecb5ee9793fd2fa4691de9a762911fd38b/src/commands/fun/modules/eco-bet.ts#L179-L196

Reply:
Good catch. @smartfrigde please move the `parseDuration` function into, say `src/lib.ts` and update all references to it accordingly. Use this function to parse the durations correctly.

Reply:
Done in latest commit